scripts/build_wheel.sh

```diff
@@ -36,6 +36,21 @@ else
     echo "Skipping store.so (not built - likely WITH_STORE is set to OFF)"
 fi
 
+if [ -f build/mooncake-store/src/libmooncake_store.so ]; then
+    echo "Copying mooncake_store.so..."
```

Comment:
I consulted @alogfans . Since the TE config is loaded during initialization and not modified dynamically, it’s fine not to use a .so here.